ctapipe/io/sst1meventsource.py

```diff
@@ -31,6 +32,10 @@ def _generator(self):
             data.count = count
             data.sst1m.fill_from_zfile_event(event, self._pixel_sort_ids)
             self.fill_R0Container_from_zfile_event(data.r0, event)
+            data.inst.subarray.tels[0] = TelescopeDescription.from_name(
```

Comment:
it might be better to pass this to the SubarrayDescription constructor instead, but this should work for existing code.  If in the future we pre-compute some things in the constructor, however, it will fail. 


e.g. this may be safer: 
```py3

event.inst.subarray = SubarrayDescription(
    name="SST1MTestBenchArray", 
    tel_positions={0: [0,0]*u.m}, 
    tel_descriptions={0:TelescopeDescription.from_name("SST-1M","DigiCam")}
)
```
or similar

Reply:
Also, using `TelescopeDescription.from_name()` of course assumes the `DigiCam.camgeom.fits.gz` and `optics.ecsv.txt`  files in your `CTAPIPE_SVC_PATH` (or ctapipe-extra package) correspond to the input file your give (i.e. the pixel numbers are correct, etc).  Since currently that was generated from Monte-Carlo, it may not be exactly right, so just keep that in mind. You might add a check in the code to make sure at least the number of pixels, etc, makes sense in case you load a file with a slightly modified camera geom.